tests/console/check_system_info.pm

```diff
@@ -91,11 +91,13 @@ sub check_nodejs_common {
     record_info('SLE-21783', 'check nodejs-common');
     zypper_call('in nodejs-common | tee /tmp/install_nodejs.log');
 
-    assert_script_run('grep -E "Installing: nodejs16" /tmp/install_nodejs.log');
-    if ((script_output(q(zypper se -i nodejs | sed -n '/nodejs16 /p' | awk '{print $1}')) !~ /i/) || \
-        (script_output(q(zypper se -i nodejs | sed -n '/nodejs-common/p' | awk '{print $1}')) !~ /i\+/)) {
-        die "Expected package is not installed";
+    my $ret = script_run(q(grep "'nodejs-common' is already installed." /tmp/install_nodejs.log));
+    if ($ret != 0) {
+        assert_script_run('grep -E "Installing: nodejs16" /tmp/install_nodejs.log');
```

Comment:
If nodejs-common hasn't installed, on what condition that the nodejs16 will be in the install log?

Reply:
https://openqa.nue.suse.com/tests/8538473#step/check_system_info/29
there's real install action

Reply:
I think only assert_script_run('grep -E "Installing: nodejs16" /tmp/install_nodejs.log'); is enough.
Since we just check the dependency relationship with nodejs-common, just after cmd to install nodejs-common. WDYT?

Reply:
Do you mean to remove the check for "i" and "i+"?  if so I'm Ok with it.

Reply:
changed the code as we discussed and now run VR again.